elementary/monitor/cli.py

```diff
@@ -123,6 +123,22 @@ def decorator(func):
             default=None,
             help="Which directory to look in for the dbt_project.yml file. Default is the current working directory.",
         )(func)
+        func = click.option(
+            "--project-profile-target",
+            type=str,
+            default=None,
+            help="Which target to load for the given profile. "
+            "If specified, the target will be used for your dbt project."
+            "Else, the --profile-target will be used.",
+        )(func)
+        func = click.option(
```

Comment:
I don't think it's critical for now but I wonder if people will expect multiple selectors to work as well.
Can be added to a future version if needed though.

Reply:
I think that dbt supports only a single selector (when passing more then one it uses the latest one), so I think that we should see the usage of it and wait for users' requests 🙂

Reply:
They support multiple:
<img width="666" alt="image" src="https://user-images.githubusercontent.com/11314965/210347881-82b37090-4909-4677-8faf-0c85c74087b2.png">

But again - I don't see it as critical for now.

Reply:
Oh I was thinking about providing multiple tags / owners.
I think it could be step 2